.github/workflows/stale.yml

```diff
@@ -0,0 +1,27 @@
+name: "Close stale issues and PRs"
+on:
+  schedule:
+    - cron:  '00 05 * * *'
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: gorzelinski/stale@add-reaction-updates
+        with:
+          repo-token: ${{ secrets.GH_BOT_TOKEN }}
+          stale-issue-label: status:stale
+          stale-issue-message: "There has been no activity in this issue for the last five years. Now it is marked as stale and will be closed in 30 days. We know it may be valid, so if you're interested in the solution, leave a comment or reaction under the issue."
+          stale-pr-message: "There has been no activity in this PR for the last five years. Now it is marked as stale and will be closed in 30 days. We know it may be valid, so if you're interested in the contribution, leave a comment or reaction under the PR."
+          close-issue-message: "We have closed the issue because of the lack of activity for the last five years. We know it might be valid, so you can reopen a similar issue if needed."
+          close-pr-message: "We have closed the PR because of the lack of activity for the last five years. We know it might be valid, so you can reopen a similar PR if needed."
+          days-before-stale: 1825
+          days-before-close: 30
+          exempt-issue-labels: support, domain:accessibility
```

Comment:
BTW, how do we know that the bot won't close a ticket that got "status:planned" label a week ago (and apart from that was dead for 10 years)? Those "status:\*" labels are added automatically when we modify the ZH board so they reflect information which is otherwise unavailable on GH.

Reply:
The `support` string covers all cases. But I can be more explicit and add those labels separately.

Reply:
It probably won't close it. The bot checks the `update_at` field - updated after the label change.

Reply:
It won't. Same situation as here: ckeditor/github-writer#205.

Reply:
Is the space character valid here?

```
support, domain:accessibility
```

Docs says:

> Comma separated list of labels that can be assigned to issues to exclude them from being marked as stale (e.g: `question,bug`)

There is no space after the comma character.

Reply:
It worked during testing, so I assume it is valid. But sticking to the docs is a good idea. Good point.